ddtrace/tracer/civisibility_payload_test.go

```diff
@@ -0,0 +1,121 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024 Datadog, Inc.
+
+package tracer
+
+import (
+	"bytes"
+	"io"
+	"strconv"
+	"strings"
+	"sync/atomic"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/tinylib/msgp/msgp"
+)
+
+func newCiVisibilityEventsList(n int) []*ciVisibilityEvent {
+	itoa := map[int]string{0: "0", 1: "1", 2: "2", 3: "3", 4: "4", 5: "5"}
+	list := make([]*ciVisibilityEvent, n)
+	for i := 0; i < n; i++ {
+		s := newBasicSpan("span.list." + itoa[i%5+1])
```

Comment:
This is a nit, but you don't really need the map to do this conversion. You could do `strconv.Itoa(i%5+1)` instead